ROADMAP.md

```diff
@@ -0,0 +1,62 @@
+# Short Term
+(getting ready for lab days)
+
+## Outcome:
+
+- The repository covers:
+    - (rust, go, nodejs, browserjs) x (ping test)
+- We have a working demonstration and a presentation for the event
+- We have a single Dashboard to track the current state of interop testing
+
+
+## EPICs
+
+- There is a clear way to tell the state of libp2p's interoperability testing
+    - Create a page that tells me which implementations are supported by our interop infrastructure,
+    - This page signal the status of each test  (not implemented / implemented / broken / passing)
+    - This page is generated automatically, nightly
+- The `lip2p/test-plans` test suite covers essential features for the demonstration
+    - browserjs
+    - nodejs
+    - webtransport
+    - webrtc
```

Comment:
I know what these are, but they are at a different level than browserjs/nodejs.

It seems like we're mixing language implementations with transports.

I do think we need to get to a place of libp2p testing across various dimensions:

<language, transport, version>

Is enumerating transports (TCP, Quic, WebTransport, WebRTC) in scope for this milestone?

I guess what's happening here is that in order to show that browser/nodejs-libp2p can talk with go-libp2p or rust-libp2p, they need to use WebTransport and/or WebRTC.

To drive clarity, I think mocking out what the page looks like will help make sure everyone is on the same page.

Reply:
Thanks for the feedback, just a quick note,

We created an epic for implementing test matrixes at https://github.com/libp2p/test-plans/issues/53
At the moment I am working on https://github.com/libp2p/test-plans/issues/55 which is step 1.